### PR TITLE
-C editorial fixes

### PR DESCRIPTION
**Pull request by paul-grassi**
Thanks @kboeckl for the graphics updates.

### CODE REVIEW

File: sp800-63c/sec3_definitions.md [hunk 1]
```diff
@@ -38,6 +38,9 @@ A defined sequence of messages between a claimant and a verifier that demonstrat
 #### Back-Channel Communication
 Communication between two systems that relies on a direct connection (allowing for standard protocol-level proxies), without using redirects through an intermediary such as a browser. This can be accomplished using HTTP requests and responses.
 
+#### Bearer Assertion
+A bearer assertion can be presented by any party as proof of the bearer’s identity. Any party in possession of the assertion can use use it as if it were their own. This type of assertion does not require the bearer to prove possession of cryptographic key material (i.e. proof-of-possession or holder-of-key).
```
Comment (jricher) [inline sp800-63c/sec3_definitions.md]: I would tweak the wording a bit here:

"An assertion presented by a party as proof of its identity, where proof of the assertion itself is sufficient proof of identity for the bearer of the assertion."

**Review by jricher — CHANGES_REQUESTED** [review]
Suggested changes to the definition of bearer assertion.

**Comment by paul-grassi** [comment]
Done!

File: sp800-63c/sec3_definitions.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ A defined sequence of messages between a claimant and a verifier that demonstrat
 Communication between two systems that relies on a direct connection (allowing for standard protocol-level proxies), without using redirects through an intermediary such as a browser. This can be accomplished using HTTP requests and responses.
 
 #### Bearer Assertion
-An assertion presented by a party as proof of its identity, where proof of the assertion itself is sufficient proof of identity for the bearer of the assertion. Any party in possession of the assertion can use use it as if it were their own. This type of assertion does not require the bearer to prove possession of cryptographic key material (i.e. proof-of-possession or holder-of-key).
+An assertion presented by a party as proof of its identity, where proof of the assertion itself is sufficient proof of identity for the bearer of the assertion.
```
Comment (jricher) [inline sp800-63c/sec3_definitions.md]: change "where proof of the assertion" to "where possession of the assertion"

**Review by jricher — APPROVED** [review]

**Pull request merged**